### PR TITLE
5.4.8 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.4.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 
 ## Release 5.4.8-7
```
Comment (Bhautik) [inline docs/release_notes_128t_5.4.md:27]: Build number is 5.4.8-8

File: docs/releases.table.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const releases = [
   {
     version: '5.4.8',
     url:'release_notes_128t_5.4#release-548-7',
```
Comment (Bhautik) [inline docs/releases.table.js:10]: build number 548-8

File: docs/about_releases.mdx [hunk 3]
```diff
@@ -23,7 +23,7 @@ Starting with version `5.4.0` the `-build` will reflect the number of iterations
 | Version | Initial GA Version | Initial GA Date | Latest GA Version |
 | --- | --- | --- | --- |
 | Release 5.5 | [5.5.3](release_notes_128t_5.5.md#release-553-4) | August 11, 2022 | [5.5.5](release_notes_128t_5.5.md#release-555-3) |
-| Release 5.4 | [5.4.4](release_notes_128t_5.4.md#release-544-9) | February 18, 2022 | [5.4.7](release_notes_128t_5.4.md#release-547-7) |
+| Release 5.4 | [5.4.4](release_notes_128t_5.4.md#release-544-9) | February 18, 2022 | [5.4.8](release_notes_128t_5.4.md#release-548-7) |
```
Comment (Bhautik) [inline docs/about_releases.mdx:26]: Date here should be October 11th

**Review by Bhautik — CHANGES_REQUESTED** [review]

File: docs/about_releases.mdx (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/about_releases.mdx:26]: this date is the initial GA date, not the current release.

**Review by Bhautik — APPROVED** [review]

**Review by Bhautik — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**